### PR TITLE
built out styles

### LINKED ISSUE #7: Define Colour and Fonts
A.C.

- Export re-useable Variable
- Dimensions as constants
- Think about the API!

### PR DESCRIPTION
**Pull request by jonathanfunk**
Closes #7

### CODE REVIEW

File: js/styles/baseStyles.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  mainBrandColor: '#FF8E3E',
+  secondaryBrandColor: '#FFDD1E',
+  tertiaryBrandColor: '#2EAEB7',
+  secondaryBtnColor: '#9DDA2C',
+  secondaryNavColor: '#FFBC8C',
+  appetizerColor: '#93B259',
+  mainCourseColor: '#37B1B0',
+  saladsColor: '#D8D8D8',
+  foodIndicatorBG: '#E1E1E1',
+  foodIndicatorNumb
```
Comment (BlakeTurner) [inline js/styles/baseStyles.js]: This key is missing a value.

**Review by BlakeTurner — CHANGES_REQUESTED** [review]
Good work.

Let's move `/js/assets` to just `/assets`

Also just one issue with a property in your styles.

**Review by BlakeTurner — APPROVED** [review]

File: js/styles/baseStyles.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { Dimensions, StyleSheet } from 'react-native';
+
+export const windowHeight = Dimensions.get('window').height;
+export const windowWidth = Dimensions.get('window').width;
+
+export const colors = {
+  mainBrandColor: '#FF8E3E',
+  secondaryBrandColor: '#FFDD1E',
+  tertiaryBrandColor: '#2EAEB7',
+  secondaryBtnColor: '#9DDA2C',
```
Comment (scottfuoco) [inline js/styles/baseStyles.js]: missing main or primaryBtnColor only have secondaryBtnColor

File: js/styles/baseStyles.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  secondaryNavColor: '#FFBC8C',
+  appetizerColor: '#93B259',
+  mainCourseColor: '#37B1B0',
+  saladsColor: '#D8D8D8',
+  foodIndicatorBG: '#E1E1E1',
+}
+
+export const typography = {
+  smallestSize: '10px',
```
Comment (scottfuoco) [inline js/styles/baseStyles.js]: Maybe add font to variable name ie. smallestFontSize

File: js/styles/baseStyles.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  headerSize: '24px',
+  fontMain: 'Ubuntu-R.ttf',
+  fontMainLight: 'Ubuntu-L.ttf',
+  fontMainMedium: 'Ubuntu-M.ttf',
+  fontMainBold: 'Ubuntu-B.ttf',
+}
+
+export default StyleSheet.create({
+  headerPrimary: {
+    fontSize: headerSize,
```
Comment (scottfuoco) [inline js/styles/baseStyles.js]: this may break, did you mean typography.headerSize.  As well as putting object references for all variable lookups.

**Review by ringong95 — APPROVED** [review]

**Pull request merged**